app/views/repositories/_repository_row_info_modal.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="modal fade" id="modal-info-repository-row" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title"><%= t('records.modal_info.head_title', record: @record.name) %></h4>
+      </div>
+
+      <div class="modal-body">
+        <p>
+          <span>
+            <%= t "records.modal_info.added_on" %>
+            <%= l @record.created_at, format: :full %>
+          </span>
+          <br>
+          <span>
+            <%= t "records.modal_info.added_by" %>
+            <%= @record.created_by.full_name %>
+          </span>
+          <% @record.repository_cells.each do |record_custom_field| %>
```

Comment:
Nazi comment :smiling_imp:
I think we are not using words like `custom_field`, `record`,... in repositories anymore, can you switch to the new wording:
- `record_custom_field` should be `repository_cell`
- `record` should be `repository_row`
...